app/models/item.rb

```diff
@@ -20,6 +20,20 @@ def self.most_revenue(quantity)
       first(quantity)
   end
 
+  def self.top_items_by_number_sold(quantity)
+    unscoped.
+      select(
+        "items.*, "\
+        "SUM(invoice_items.quantity) AS most_items"
+      ).
+      joins(:transactions).
+      where("transactions.result = 'success'").
+      joins(:invoice_items).
```

Comment:
@pwentz I believe you can do this as one joins command -> `joins(:transactions, :invoice_items)`